### DISCUSSION TITLE
112

### DISCUSSION THREAD

**Discussion by giscus** [tw93/Weekly#35] [General]
# 潮流周刊第112期 - 咖啡店猫

封面来源于周末下午去木星咖啡喝澳白敲代码，睡在我旁边椅子的猫，后面走的时候给拍了一张，很是可爱漂亮，比我家汤圆可乐大了不少，一样过得无忧无虑。

https://weekly.tw93.fun/posts/112-%E5%92%96%E5%95%A1%E5%BA%97%E7%8C%AB

**Comment by Alice-smail** [comment]
我想问下，我用Pake下载的mac版本的chatGpt app，为什么登录的时候会一直报错呢，Route Error (400 ): "Unknown error"
